apps/portals/tsconfig.json

```diff
@@ -8,7 +8,9 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noFallthroughCasesInSwitch": true,
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "moduleResolution": "bundler",
```

Comment:
`"bundler"` is one of the values for `moduleResolution` permitted by Vite 5. It makes the most sense for all of the packages in this repo (the alternative is `'node16'`/`'nodenext'`)